### PR TITLE
added withdrawals docs

### PR DESCRIPTION
**Pull request by zehemz**
- Added missing withdrawal metadata docs.
- Added missing withdrawal endpoint docs.
- Added new error code documentation.

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -24,6 +24,12 @@ point of sale systems, and much more. Below you will find details on how the
 system functions, along with examples in common programming languages.
 
 # Changelog
+
+### 2022-08-18
+Add new error code '0506'
+Added withdrawal new endpoints documentation.
+Added withdrawal methodds new endpoints documentation.
```
Comment (thenewversion) [inline source/index.html.md:31]: You may want to add the deprecation notices here as well

**Review by thenewversion — COMMENTED** [review]
Nice!!
That was long overdue but good to see it now. Only concern I may have is that we have marked some endpoints as deprecated but we need to give an ETA as to when they will be closed down completely if possible.

File: source/index.html.md [hunk 2]
```diff
@@ -2139,9 +2146,78 @@ Field Name | Type | Description | Units
 **account_identifier_name** | String | Account identifier name to fund with the specified currency. | -
 **account_identifier** | String | Identifier to where the funds can be sent to. | -
 
+## Withdrawal taxonomy
+
+In order to execute withdrawals is it mandatory to know under which available method, network and protocol you want to operate.
```
Comment (MelisaSosa) [inline source/index.html.md:2151]:
```suggestion
In order to execute withdrawals is it mandatory to know under which available method, network, asset and protocol you want to operate.
```

**Review by MelisaSosa — COMMENTED** [review]
LG, small detail

**Comment by zehemz** [comment]
@thenewversion about deprecation date is not yet planned, first of all a friendly notice is enough, second stage is about to see how much traffic there is and execute a migration plan. We have wrote down this https://bitsomx.atlassian.net/wiki/spaces/ARC/pages/3403055265/API+Endpoint+Deprecation+Process

**Review by MelisaSosa — APPROVED** [review]

**Review by thenewversion — APPROVED** [review]
LGTM

**Pull request merged**